lib/migration_helper.rb

```diff
@@ -1,54 +0,0 @@
-# Utility functions used in several migrations
```

Comment:
Wait, i was wrong. this can be deleted, as @cfeeney-va is correct. What happened is initially some tasks for setting up the skills translator were done as migrations instead of rake tasks. They were more recently changed to be rake tasks, so this MigrationHelper is no longer necessary.